components/eam/src/physics/cam/gw_convect.F90

```diff
@@ -31,19 +30,20 @@ module gw_convect
 
 !==========================================================================
 
-subroutine gw_convect_init(k700_in, mfcc_in, errstring)
-  ! Index at 700 mb.
-  integer, intent(in) :: k700_in
-  ! Source spectra to keep as table.
-  real(r8), intent(in) :: mfcc_in(:,:,:)
-  ! Report any errors from this routine.
-  character(len=*), intent(out) :: errstring
-
+subroutine gw_convect_init( plev_src_wind, mfcc_in, errstring)
+  use ref_pres, only: pref_edge
+  real(r8), intent(in) :: plev_src_wind        ! previously hardcoded to 70000._r8
```

Comment:
I'd add here in the comments that plev_src_wind should be in units Pa to match pref_edge.